docs/reference.md

```diff
@@ -207,12 +207,12 @@ For the `whisk.error()` you can return a rejected Promise (i.e. Promise.reject).
 ### JavaScript runtime environments
 
 JavaScript actions can be executed in Node.js version 6 or Node.js version 8.
-Currently actions are executed by default in a Node.js version 6.12.2 environment.  
+Currently actions are executed by default in a Node.js version 6.12.2 environment.
```

Comment:
Should also version bump to 6.14.0 here?

Reply:
Yes thank you

Reply:
@dgrove-oss I made it generic just 6 since below it talks about the specific version